frontends/qt/main.cpp

```diff
@@ -163,6 +163,11 @@ int main(int argc, char *argv[])
     qputenv("DRAW_USE_LLVM", "0");
 #endif
 
+    // Force software rendering over GPU-accelerated rendering as various rendering artefact issues
+    // were observed on Windows and the app crashes on some Linux systems.
```

Comment:
So, none of the other values worked, not even OpenVG?

Reply:
Or RHI:

> From Qt 5.14 onwards, the default adaptation gains the option of rendering via a graphics abstraction layer, the Qt Rendering Hardware Interface (RHI), provided by the QtGui module.

(from https://doc.qt.io/qt-5/qtquick-visualcanvas-scenegraph-renderer.html#rendering-via-the-qt-rendering-hardware-interface)

Reply:
RHI didn't do anything unfortunately.

OpenVG I missed - do you want to try to quickly enable it in a branch based off of v4.29.0? 

In the PR you can remove the `- if defined APPVEYOR_PULL_REQUEST_NUMBER appveyor exit` line from appveyor to get a Windows build artefact, which we could test.

Reply:
So, tried `openvg` - doesn't work due to a "Could not create scene graph context for backend 'openvg' - check that plugins are installed correctly in c:/program files/bitbox/plugins". We don't even have plugins and openvg seems to dying: https://www.phoronix.com/scan.php?page=news_item&px=Qt-5.9-OpenVG-Overview

RHI doesn't work due to https://bugreports.qt.io/browse/QTBUG-78682, I believe. QtWebEngine is based on chromium which requires [ANGLE](https://doc.qt.io/qt-5/windows-requirements.html) lib. Otherwise, RHI results in a black screen.

We're out of options.

Reply:
> We're out of options.

Well, I imagine a more involved future option is to build Qt from sources, linking with ANGLE lib.